core/client/client_ip.go

```diff
@@ -96,6 +103,17 @@ func (c *IPClient) measureClockOffsetIP(ctx context.Context, log *zap.Logger, mt
 		log.Error("failed to enable timestamping", zap.Error(err))
 	}
 
+	var KEData ntske.Data
+	if c.Auth.Enabled {
+		KEData, err = c.Auth.NTSKEFetcher.FetchData()
+		if err != nil {
+			log.Error("failed to fetch key exchange data", zap.Error(err))
```

Comment:
In the SCION client we just log this (as Info) and proceed in sending the request unauthenticated. I would do it the same way here, for now.

Reply:
Done